stix2/sources/__init__.py

```diff
@@ -340,62 +353,67 @@ def __init__(self):
         self.data_sources = {}
```

Comment:
`name` was removed so it should be taken out of the docstring, too.

Reply:
good catch, removed